src/client/swagger/unity.yml

```diff
@@ -1162,55 +1168,26 @@ components:
         id:
           description: the idpe id of the user
           type: string
+        # TODO(ariel): find out if these are account or user specific
         firstName:
           type: string
+        # TODO(ariel): find out if these are account or user specific
         lastName:
           type: string
         email:
+          # TODO(ariel): find out about this one
+          description: Not sure if this is the billing account email or the account email, but this might be unnecessary since that should be included in the Account or nested in the Account
           type: string
-        orgId:
```

Comment:
Removed orgId because that should already be accessible by the time the user accesses this endpoint. Meaning, they will be authenticated and the data can be obtained by IDPE

Reply:
🤔 I'm not sure I follow, what're "claims" in the session cookie? 

re: reducing the number of API calls to IDPE to obtain the orgID - when a user is authenticated, one of the first API requests we make is for the orgID (among a few other requests). In this case, we won't be adding on another request, simply taking advantage of the existing flow

Reply:
The session cookie is a JWT, and the contents of a JWT are called "claims".  As long as the API request returns the orgID that's in the session cookie, that should be fine.